code/html/spec.html.template

```diff
@@ -0,0 +1,28 @@
+@@layout("application", { "title": "TOML: {{ language }} {{ version }}" })
+
+@@include("{{ locale }}/nav.html")
+
+<div class="spec" data-controller="nav" data-action="scroll@window->nav#scroll">
+  <div class="lg:flex pt-14">
+    <aside class="w-full bg-white lg:w-64 lg:pr-64 lg:block">
+      <div class="fixed bg-white z-20 left-0 right-0 mt-3 pl-4 py-4 border-b lg:hidden" data-target="nav.menuLink">
+        <div class="relative">
+          <a href="#" class="block font-semibold lg:hidden text-orange-800" data-action="click->nav#onMenuClick">Menu</a>
+          <div class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700">
+            <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z"></path></svg>
+          </div>
+        </div>
+      </div>
+
+      <nav class="hidden fixed w-full overflow-y-scroll top-0 bottom-0 bg-white z-10 py-32 pb-8 px-4 leading-snug lg:w-68 lg:block lg:pl-8 lg:py-24" data-target="nav.menu">
```

Comment:
I feel like a bit more padding at the bottom would be nice, now that there's no footer. 🙈